plugins/foreman_salt/13.1/index.md

```diff
@@ -175,9 +175,16 @@ In `/etc/salt/foreman.yaml`, make the following changes:
     :timeout:  10
     :salt:  /usr/bin/salt
     :upload_grains:  true
+    :filecache: false
 
 If your Smart Proxy uses SSL, then the certs and key configured in the YAML should be the same ones it uses to talk to Foreman. If you're already using Puppet in Foreman, consult `/etc/puppet/foreman.yaml` or `/etc/puppet/node.rb` for those settings.
 
+The :filecache: option makes foreman-node read the grains cache from files rather than using salt-run. 
+
+Please note this will only work if the master is not configured to use an external minion cache module. With salt's default configuration the filecache option works fine and should be used as it is much faster than getting the cache using salt-run. Please note that for this to work, the "msgpack" rubygem needs to be present.
```

Comment:
Is that something one has to do manually or is it brought in as a dependency when installing the rpm?

Reply:
That I am not sure of. Currently, in our containerized foreman proxy we are installing it manually using gem install because there's no package available. @sbernhard commited something to foreman-packaging though adding the rubygem, but as of now, there's no dependency in the spec file yet.